ajax/libs/webvr-polyfill/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "webvr-polyfill",
+  "description": "Use WebVR today, on mobile or desktop, without requiring a special browser build.",
+  "filename": "webvr-polyfill.min.js",
+  "license": "Apache-2.0",
+  "homepage": "https://immersive-web.github.io/webvr-polyfill/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/immersive-web/webvr-polyfill"
+  },
+  "authors": [
+    "Boris Smus <boris@smus.com>",
+    "Brandon Jones <tojiro@gmail.com>",
+    "Jordan Santell <jordan@jsantell.com>"
+  ],
+  "keywords": [
+    "vr",
+    "webvr",
+    "webxr",
+    "polyfill"
+  ],
+  "autoupdate": {
+    "source": "git",
```

Comment:
New versions are not being tagged on git, please use an npm auto-update config here: https://www.npmjs.com/package/webvr-polyfill